operations/mimir-mixin/dashboards/compactor.libsonnet

```
@@ -40,6 +162,106 @@ local utils = import 'mixin-utils/utils.libsonnet';
           |||
         ),
       )
+      .addPanel(
+        $.panel('Longest time since last successful run') +
+        $.panelDescription(
+          'Longest time since last successful run',
+          |||
+            Displays the amount of time since the most recent successful execution
+            of the compactor.
+            The value shown will be for the compactor replica that has the longest time since its
+            last successful run.
+            The table to the right shows a summary for all compactor replicas.
+
+            If there is no time value, one of the following messages might appear:
+
+            - If you see "No compactor data" in this panel, that means that no compactors are active yet.
+
+            - If you see "No successful runs" in this panel, that means that compactors are active, but none 
+              of them were successfully executed yet.
+
+            These might be expected - for example, if you just recently restarted your compactors, they might not 
+            yet be broadcasting metrics, or they might not have had a chance to complete their first compaction run. 
```

There's no "broadcast" of metrics in Prometheus, since it's a pull based system. Also, metrics are exposed right after startup, so I would just remove it.

```suggestion
            These might be expected - for example, if you just recently restarted your compactors,
            they might not have had a chance to complete their first compaction run. 
```